### PR TITLE
Add length member to event_stream_message

### PR DESCRIPTION
**Pull request by marcomagdy**
Needed when serializing the message bits to a network stream for
example.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by marcomagdy** [comment]
The failure in the msvc CI doesn't make any sense given my changes. Either it's always been broken or it's a fluke.

File: include/aws/event-stream/event_stream.h [hunk 1]
```diff
@@ -43,6 +43,7 @@ struct aws_event_stream_message {
     struct aws_allocator *alloc;
     uint8_t *message_buffer;
     uint8_t owns_buffer;
+    size_t len;
```
Comment (JonathanHenson) [inline include/aws/event-stream/event_stream.h]: There's already a len field

aws_event_stream_length()

Comment (JonathanHenson) [reply include/aws/event-stream/event_stream.h]: https://github.com/awslabs/aws-c-event-stream/blob/master/include/aws/event-stream/event_stream.h#L185

**Comment by marcomagdy** [comment]
Turns out there's `aws_event_stream_message_total_length`.
Closing this PR

**Pull request closed without merging**